### DISCUSSION TITLE
Prevent the first admin from suspending their own account?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13291] [Archive]
I didn't do it, but Is it wise that the first admin user can suspend their own account? Maybe disable this unless there's more than one admin account?

**Comment by benhaynes** [comment]
Do you mean from failed login attempts? Not sure about removing security because you only have a single user. You can adjust allowed failed login attempts within settings, and the project owner could reset the locked account using the database or CLI.

**Reply by sxsws** [reply]
In the users section, click on admin account, then under admin options there's the option to suspend the account. there's even the option of 'draft' which doesn't really make sense?

**Reply by benhaynes** [reply]
I thought we already had a check-in place that you can't deactivate the last admin, right @rijkvanzanten ?

Also, `draft` is if you're creating a user account that you don't want to be live yet.

**Reply by azrikahar** [reply]
> I thought we already had a check-in place that you can't deactivate the last admin

Yeah we definitely have it. But although it does prevent us from deleting or removing admin role from the last admin, the current check does seem to not include status changes so "set status as suspended" action can technically still go through successfully. Added a fix in #13309.

**Reply by sxsws** [reply]
the directus community is excellent!
